modules/meta.analysis/R/run.meta.analysis.R

```diff
@@ -7,7 +7,7 @@
 # http://opensource.ncsa.illinois.edu/license.html
 #-------------------------------------------------------------------------------
 
-run.meta.analysis.pft <- function(pft, iterations, random = TRUE, threshold = 1.2, dbfiles, dbcon, use_ghs = TRUE) {
+run.meta.analysis.pft <- function(pft, iterations, random = TRUE, threshold = 1.2, dbfiles, dbcon, use_ghs = TRUE, update) {
```

Comment:
would be good to have a default on `update`

Reply:
Hi @mdietze, I think the default value of `update` should be `FALSE`, So whenever `TRUE` is passed as an argument, the `run.meta.analysis` will run. Am I right?